src/test/scala/com/databricks/labs/overwatch/utils/StringExtTest.scala

```diff
@@ -27,11 +29,11 @@ class StringExtTest extends AnyFunSpec {
     }
 
     it("String with white % characters") {
-      assert(!"""Business %Name""".containsNoSpecialChars)
+      assert("""Business %Name""".containsNoSpecialChars)
     }
 
     it("String with white $ characters") {
-      assert(!"""Business $Name""".containsNoSpecialChars)
+      assert("""Business $Name""".containsNoSpecialChars)
     }
   }
 }
```

Comment:
did you mean to leave these changes or are these left-overs from the previous PR? Please drop these changes if they were unintentional.

Reply:
done